apps/vela-mobile/src/css/quasar.variables.scss

```
@@ -0,0 +1,12 @@
+$primary: #5b4af7;
+$secondary: #e8447a;
+$accent: #1db87a;
+$warning: #f59e0b;
+$error-color: #f04b5a;
+
+$dark: #11101e;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)